### ISSUE TITLE
dart linter error: Record field names can't be private.

### ISSUE THREAD

**Issue by bobwith2bees** [inveker/web3dart_builders#12]
Ok, I'm getting my head wrapped around records and how to call them, but I think I stumbled upon an issue .

What I am trying to do is take the public ENS contracts from https://github.com/ensdomains/ens-contracts/tree/mainnet/deployments/mainnet and use web3dart_builders to generate code I can call to look up ENS names.

I have the abi's properly named and generating .g.dart files.  pubspec.yaml has 
```environment:
  sdk: ">=3.0.0 <4.0.0"

  # other basic entries I didn't think relevant

dev_dependencies:
  build_runner: ^2.4.8  
  web3dart_builders: ^0.1.0
  flutter_lints: ^3.0.1
```

I see a dart linter error "Record field names can't be private."  

The abi file has an entry like this one for _dnssec [full file link](https://github.com/ensdomains/ens-contracts/blob/b2942eeee645e202fa31d268594343fed7e9a169/deployments/mainnet/DNSRegistrar.json#L18)
```
        {
          "internalType": "contract DNSSEC",
          "name": "_dnssec",
          "type": "address"
        },
```

The linter complains about the _dnssec being private here:

```dart
  Future<String> setOracle(
    ({_i1.EthereumAddress _dnssec}) args, { 
    required _i1.Credentials credentials,
    _i1.Transaction? transaction,
  }) async {
    final function = self.abi.functions[6];
    assert(checkSignature(function, '7adbf973'));
    final params = [args._dnssec];
    return write(
      credentials,
      transaction,
      function,
      params, 
    );
  }
```

FYI - The public resolver and universal resolver are generating additional errors, but I figured I would start with one bug to see if just need to enable/disable some features.

**Comment by inveker** [comment]
@bobwith2bees Hello. Yes, you are indeed right. Update required. The only question is how to do it so that it works. I would not like to change the names of the arguments passed from abi in any way (for example, remove _).

**Comment by bobwith2bees** [comment]
@inveker Understood.   It's not a showstopper for me; just wanted to log for visibility.  

A related naming challenge to unit test the solution. 

This file has multiple functions called "resolve" definitions that result in a number of `'The name 'Resolve' is already defined.'` and similar already-defined errors.    
https://github.com/ensdomains/ens-contracts/blob/b2942eeee645e202fa31d268594343fed7e9a169/deployments/mainnet/UniversalResolver.json#L239

**Comment by inveker** [comment]
Fixed in 0.1.2
